agentpluginapi/i_linux_agent_command_builder.py

```diff
@@ -58,11 +58,17 @@ def build_run_command(self, run_options: LinuxRunOptions):
     @abc.abstractmethod
     def get_command(self) -> str:
         """
-        Gets the resulting command
+        Joins built commands into a one-liner command and returns it
+        """
+
+    @abc.abstractmethod
+    def get_command_list(self) -> List[str]:
+        """
+        Returns the list of built commands
         """
 
     @abc.abstractclassmethod
-    def reset_command(self):
```

Comment:
I think this was intentional. Just `reset` is not really clear what it is resetting.

Reply:
Additionally it's consistent. All the methods are "build_this_command", "build_that_command", "get_command". So "reset_command" follows the same scheme.